core/amber/src/main/scala/edu/uci/ics/backenderror/Error.scala

```diff
@@ -0,0 +1,13 @@
+package edu.uci.ics.backenderror
```

Comment:
do we just put it under `edu.uci.ics`? maybe `edu.uci.ics.amber`?

Reply:
package name... don't think it's a good idea to put it directly under `edu.uci.ics`